returnn/training.py

```diff
@@ -237,6 +238,37 @@ def _get_run_cmd(self):
 
         return run_cmd
 
+    def info(self):
+        def try_load_lr_log(file_path: str):
```

Comment:
you have now removed the "try" part from the `try_load_lr_log` function completely. (If you keep it this way, remove the try part from the name).

But I am wondering if in some cases this will fail (e.g. training just startet so no LR file will be present) and if the manager then crashes..?

Reply:
Yeah leaving the check in is probably best.